appengine/cloud-tasks/spec/task_spec.rb

```diff
@@ -16,23 +16,25 @@
 require "rspec"
 require "rack/test"
 require "google/cloud/tasks"
+require 'cgi'
```

Comment:
bit nit-picky, but could this be double quotes?

Reply:
done.